### PR TITLE
Wave Sort

### PR DESCRIPTION
**Pull request by sayantani11**
Sorry for being late
Here's issue #184

### CODE REVIEW

File: Algorithms/Sorting/Wave_Sort.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   
+    numbers=[]
+
+    n=int(input("Length of arr: "))
+
+    print("Enter the elements: ")
+
+    for i in range(n):
+        elements=int(input())
+        numbers.append(elements)
```
Comment (rohanbabbar04) [inline Algorithms/Sorting/Wave_Sort.py:40]: Use list map instead of using a for loop

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

**Comment by sayantani11** [comment]
@rohanbabbar04 done!

**Comment by sayantani11** [comment]
@rohanbabbar04 done!

File: Algorithms/Sorting/Wave_Sort.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        if i>0 and arr[i-1]>arr[i]:
+            swap(arr,i,i-1)
+        
+        if i<n-1 and arr[i]<arr[i+1]:
+            swap(arr,i,i+1)
+
+    return arr
+
+
+
```
Comment (rohanbabbar04) [inline Algorithms/Sorting/Wave_Sort.py:25]: Remove extra lines

File: Algorithms/Sorting/Wave_Sort.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# Driver code for the function:
+
+if __name__=='__main__':
+
```
Comment (rohanbabbar04) [inline Algorithms/Sorting/Wave_Sort.py:30]: remove extra line

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]
Add sample I/O at the end of the code

**Comment by sayantani11** [comment]
@rohanbabbar04 is it okay?

File: Algorithms/Sorting/Wave_Sort.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# As we understand from the name sort it is a way of sorting that makes the 
+#    list of element look like a wave
+
+# For example an array after a wave sort will be:
+#    >=arr[1]<=arr[2]>=arr[3]<=
+
+# SAMPLE INPUT:
+#    10, 20, 5, 17, 46, 35, 29, 59, 70
+
```
Comment (rohanbabbar04) [inline Algorithms/Sorting/Wave_Sort.py:9]: Use docstrings instead of #

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

**Comment by sayantani11** [comment]
@rcoder23 The issue isn't reviewed yet

**Comment by rohanbabbar04** [comment]
@sayantani11 You have still not used docstrings (''') 
Use ''' for multiple line comment instead of #

**Review by rohanbabbar04 — APPROVED** [review]

**Comment by sayantani11** [comment]
@rcoder23 please merge

File: Algorithms/Sorting/Wave_Sort.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
+''' As we understand from the name sort it is a way of sorting that makes the 
+list of element look like a wave
+
+  For example an array after a wave sort will be:  
+   >=arr[1]<=arr[2]>=arr[3]<= 
+
+ SAMPLE INPUT:
```
Comment (rcoder23) [inline Algorithms/Sorting/Wave_Sort.py:7]: **Length of array:** statement missing

File: Algorithms/Sorting/Wave_Sort.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+''' Driver code for the function: '''
+if __name__=='__main__':
+
+    numbers=[]
+
+    n=int(input("Length of array: "))
+
+    print("Enter the elements: ")
+
+    numbers = list(map(int, input().split()))
```
Comment (rcoder23) [inline Algorithms/Sorting/Wave_Sort.py:35]: Error in the code , can be correct by 
**numbers = list(map(int, input().split(",")))**

**Review by rcoder23 — CHANGES_REQUESTED** [review]
Try to add input and output statement together

**Comment by sayantani11** [comment]
@rcoder23 Please review

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**